inc/cpt.php

```diff
@@ -127,7 +127,7 @@ function udb_widget_columns( $column, $post_id ) {
 			$widget_type = get_post_meta( $post_id, 'udb_widget_type', true );
 
 			if ( ! $widget_type ) {
-				do_action( 'udb_compat_widget_type', $post_id );
+				do_action( 'udb_compat_widget_type', $post_id ); // do we need a backwards compatibility check here as we move the settings over in backwards-compatibility.php on init
```

Comment:
The empty `udb_widget_type` possibility is open here.. sample case:
Users update the plugin, and compatibility check running, and then writing the compatibility meta.

Then turned out, they have some widgets to import (in this case, the compat meta had been written). If we skip checking, then `udb_widget_type` remains empty.